### PR TITLE
fix postgres syntax error during migration

### PR DESCRIPTION
**Pull request by WnP**
when doing migration using bee tool and postgresql as database postgresql return the following error:

```
pq: syntax error at or near "`"
```
This patch fix this error

### CODE REVIEW

File: migration/migration.go [hunk 1]
```diff
@@ -101,10 +101,17 @@ func (m *Migration) Exec(name, status string) error {
 }
 
 func (m *Migration) addOrUpdateRecord(name, status string) error {
+	var err error
+	var p orm.RawPreparer
 	o := orm.NewOrm()
 	if status == "down" {
 		status = "rollback"
-		p, err := o.Raw("update migrations set `status` = ?, `rollback_statements` = ?, `created_at` = ? where name = ?").Prepare()
```
Comment (astaxie) [inline migration/migration.go]: maybe we just remove the `  for mysql. as it's not necessary

**Comment by WnP** [comment]
@astaxie you're right, fixed in last commit

**Comment by WnP** [comment]
@astaxie I'm going to make a new pull request because we don't need 3 commits to do something simple like that

**Comment by WnP** [comment]
see #1434

**Pull request closed without merging**